src/util.ts

```diff
@@ -27,7 +27,8 @@ export const getCondaArch = () => {
     'linux-x64': 'linux-64',
     'linux-arm64': 'linux-aarch64',
     'linux-ppc64': 'linux-ppc64le',
-    'win32-x64': 'win-64'
+    'win32-x64': 'win-64',
+    'win32-arm64': 'win-arm64'
```

Comment:
as long as https://github.com/mamba-org/micromamba-releases/releases/tag/2.1.0-0 doesn't have binaries for win-arm64, this won't work

Reply:
Ah yes, and I guess we'll need to create `win-arm64` builds on https://github.com/conda-forge/micromamba-feedstock first, xref https://github.com/conda-forge/conda-forge.github.io/issues/1940.